src/NzbDrone.Core/Backup/BackupService.cs

```diff
@@ -205,9 +205,10 @@ private void BackupConfigFile()
 
         private void CreateVersionInfo()
         {
-            var builder = new StringBuilder();
+            var tempFile = Path.Combine(_backupTempFolder, "VERSION");
```

Comment:
If we're going to include non-version info then I don't think naming it `VERSION` makes sense.

Reply:
Maybe `INFO` would be better?